### DISCUSSION TITLE
Training metrics explaination

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [tracel-ai/burn#2411] [Q&A] [answered]
Hi,
This is the dashboard displayed when training mnist dataset. What does the percentage mean in the red rectangle ?
![image](https://github.com/user-attachments/assets/2e5f7280-5c62-4e0c-8aa3-7da8bea7aa10)

**Comment by laggui** [comment, marked as answer]
The metrics are computed for each batch and aggregated for all items in the dataset (completed during a single epoch).

That's why the dashboard reports the metric evaluated for the current batch and the metric evaluated the whole dataset (seen during the last complete epoch).

**Comment by wangjiawen2013** [comment]
The following is another snapshot. It seems that the training (red color) and validation (blue color) were performed by turns. Why weren't them performed synchronously ?

![企业微信截图_17314810121730](https://github.com/user-attachments/assets/0055925c-5068-4a94-ad43-e70e98ddc84e)

**Reply by laggui** [reply]
The training and validation are indeed perform in turns! At the end of each training epoch, the trained model is evaluated on the validation set. When the validation is completed, the next training epoch starts.
